tests/openvino/native/quantization/test_quantizer_config.py

```diff
@@ -13,177 +13,38 @@
-                 NodeWithType('Conv_1', OVConvolutionMetatype),
-                 NodeWithType('Output_1', OutputNoopMetatype),
-                 ]
-        node_edges = [('Input_1', 'Conv_1'), ('Conv_1', 'Output_1')]
-        original_mock_graph = create_mock_graph(nodes, node_edges)
-        self.nncf_graph = get_nncf_graph_from_mock_nx_graph(original_mock_graph)
-
-
-class NNCFGraphToTestDepthwiseConv:
-    def __init__(self):
-        #       Original graph
-        #          Input_1
-        #             |
-        #        DepthwiseConv_1
-        #             |
-        #           Output_1
-        nodes = [NodeWithType('Input_1', InputNoopMetatype),
-                 NodeWithType('Conv_1', OVDepthwiseConvolutionMetatype),
-                 NodeWithType('Output_1', OutputNoopMetatype),
-                 ]
-        node_edges = [('Input_1', 'Conv_1'), ('Conv_1', 'Output_1')]
-        original_mock_graph = create_mock_graph(nodes, node_edges)
-        self.nncf_graph = get_nncf_graph_from_mock_nx_graph(original_mock_graph)
-
-
-@pytest.mark.parametrize('nncf_graph', [NNCFGraphToTest()])
-def test_default_quantizer_config(nncf_graph):
-    algo = PostTrainingQuantization(PostTrainingQuantizationParameters())
-    min_max_algo = algo.algorithms[0]
-    min_max_algo._backend_entity = OVMinMaxAlgoBackend()
-    q_setup = min_max_algo._get_quantizer_setup(nncf_graph.nncf_graph, GraphPattern())
-
-    weight_default_config = QuantizerConfig(mode=QuantizationMode.SYMMETRIC,
-                                            num_bits=8,
-                                            signedness_to_force=True,
-                                            per_channel=True)
-    activation_default_config = QuantizerConfig(mode=QuantizationMode.SYMMETRIC,
-                                                num_bits=8,
-                                                signedness_to_force=None,
-                                                per_channel=False)
-
-    for quantization_point in q_setup.quantization_points.values():
-        if quantization_point.is_weight_quantization_point():
-            assert quantization_point.qconfig == weight_default_config
-        if quantization_point.is_activation_quantization_point():
-            assert quantization_point.qconfig == activation_default_config
-
-
-@pytest.mark.parametrize('weight_granularity', [Granularity.PERCHANNEL, Granularity.PERTENSOR])
-@pytest.mark.parametrize('activation_granularity', [Granularity.PERTENSOR])
-@pytest.mark.parametrize('preset', [QuantizationPreset.MIXED, QuantizationPreset.PERFORMANCE])
-@pytest.mark.parametrize('weight_bits', [8])
-@pytest.mark.parametrize('activation_bits', [8])
-@pytest.mark.parametrize('signed_weights', [None])
-@pytest.mark.parametrize('signed_activations', [None])
-@pytest.mark.parametrize('nncf_graph', [NNCFGraphToTest()])
-# TODO(l-bat): add signed_activations and signed_weights which should be independent from HW config.
```

Comment:
TODO missed?

Reply:
No, it's here https://github.com/openvinotoolkit/nncf/pull/1471/files#diff-ef78d8cd87a574c2ac9a393306a70d06c46317306132e4b6ce26dbb72edf7deeR95